{
  "artifact_3425547677": {
    "java_home": "extracted/jdk-26",
    "java_version_output": "openjdk 26-internal 2026-03-17\nOpenJDK Runtime Environment (build 26-internal-lmesnik-a6a9f01c1496ab231d56132cac78d21a7ca03719)\nOpenJDK 64-Bit Server VM (build 26-internal-lmesnik-a6a9f01c1496ab231d56132cac78d21a7ca03719, mixed mode, sharing)"
  },
  "status": "passed",
  "history": [],
  "datetime": "2025-06-29T18:49:19.384362+00:00",
  "url": "https://github.com/openjdk/jdk/pull/26031",
  "number": 26031,
  "title": "8359366: RunThese30M.java EXCEPTION_ACCESS_VIOLATION in JvmtiBreakpoints::clearall_in_class_at_safepoint",
  "head_sha": "a6a9f01c1496ab231d56132cac78d21a7ca03719",
  "run_url": "https://github.com/dougxc/openjdk-pr-canary/actions/runs/15958228788"
}